wizard/WizardCreateWallet2.qml

```diff
@@ -49,39 +100,353 @@ Rectangle {
+                tooltipIconVisible: true
+                Accessible.role: Accessible.StaticText
+                Accessible.name: qsTr("Recovery phrase (mnemonic seed)") + translationManager.emptyString;
+                KeyNavigation.up: header
+                KeyNavigation.backtab: header
+                Keys.onDownPressed: header.focusOnListGrid()
+                Keys.onTabPressed: header.focusOnListGrid()
+            }
+
+            ColumnLayout {
+                id: seedListGridColumn
 
-            WizardAskPassword {
-                id: passwordFields
+                function clearFields() {
+                    for (var i = 0; i < wizardCreateWallet2.hiddenWords.length; i++) {
+                        seedListGridColumn.children[0].children[wizardCreateWallet2.hiddenWords[i]].wordText.visible = true;
+                        seedListGridColumn.children[0].children[wizardCreateWallet2.hiddenWords[i]].lineEdit.text = "";
+                        seedListGridColumn.children[0].children[wizardCreateWallet2.hiddenWords[i]].lineEdit.readOnly = false;
+                    }
+                }
+            }
+
+            RowLayout {
+                Layout.fillWidth: true
+                Layout.alignment: Qt.AlignRight
+
+                Timer {
+                    id: checkSeedListGridDestruction
+                    interval: 100; running: false; repeat: true
+                    onTriggered: {
+                        if (!wizardCreateWallet2.seedListGrid) {
+                            var newSeedListGrid = Qt.createComponent("SeedListGrid.qml");
+                            wizardCreateWallet2.seedListGrid = newSeedListGrid.createObject(seedListGridColumn);
+                            appWindow.showStatusMessage(qsTr("New seed generated"),3);
+                            pageRoot.forceActiveFocus();
+                            checkSeedListGridDestruction.stop();
+                        }
+                    }
+                }
+
+                MoneroComponents.StandardButton {
+                    id: createNewSeedButton
+                    visible: appWindow.walletMode >= 2
+                    small: true
+                    primary: false
+                    text: qsTr("Create new seed") + translationManager.emptyString
+                    tooltip: qsTr("Generate a new seed") + translationManager.emptyString
+                    onClicked: {
+                        wizardController.restart(true);
+                        wizardController.createWallet();
+                        wizardCreateWallet2.seedArray = wizardController.walletOptionsSeed.split(" ")
+                        wizardCreateWallet2.seedListGrid.destroy();
+                        checkSeedListGridDestruction.start();
+                    }
+                    Accessible.role: Accessible.Button
+                    Accessible.name: qsTr("Generate a new seed") + translationManager.emptyString
```

Comment:
Would change it to "Create new seed" so that we don't have to two similar strings for no apparent reason.